services/discourse/discourse.service.js

```diff
@@ -13,6 +13,10 @@ const schema = Joi.object({
   like_count: nonNegativeInteger,
 }).required()
 
+const queryParamSchema = Joi.object({
+  hostUrl: optionalUrl.required(),
```

Comment:
I don't feel strongly, though another option for naming this parameter might be `server`.

Reply:
I like that better 👍 I struggled a bit naming these, so I'm not attached to `hostUrl` either